cpp/src/Ice/OutgoingAsync.cpp

```diff
@@ -623,9 +633,9 @@ ProxyOutgoingAsyncBase::runTimerTask()
     }
 }
 
-OutgoingAsync::OutgoingAsync(const ObjectPrxPtr& prx, bool synchronous) :
-    ProxyOutgoingAsyncBase(prx),
-    _encoding(getCompatibleEncoding(prx->_getReference()->getEncoding())),
+OutgoingAsync::OutgoingAsync(const Ice::ObjectPrxPtr& proxy, bool synchronous) :
```

Comment:
Why add the `Ice::` qualifier? We're using the `Ice` namespace already.

Reply:
removed